detox/src/devices/android/Emulator.js

```diff
@@ -24,6 +24,7 @@ class Emulator {
   }
 
   async boot(emulatorName, options = {port: undefined}) {
+    const userLaunchArgs = (argparse.getArgValue('deviceLaunchArgs') || '').split(/\s+/);
```

Comment:
Most likely, you should extend `options` with `userLaunchArgs` property too, similar to AppleSimUtils.js. And then you would get args from the command line/env var on the level of DeviceDriver.

Reply:
It is a suggestion though, not a mandatory thing to do.